src/Apps/Artwork/Components/__tests__/OtherWorks.test.tsx

```diff
@@ -18,9 +18,7 @@ describe("OtherWorks", () => {
       id: "artwork1",
       _id: "asdbsd",
       layers: [],
-      layer: {
-        artworksConnection: null,
-      },
+      layer: null,
```

Comment:
This replicates what you see when there's an error resolving the `layer` field. Without the above fix we get errors in the specs, and with the fix 🍏

Reply:
Although, rather than update this base fixture, might be better to just add an entire specific test for this. One sec...